src/Agent/NewRelic/Profiler/ConfigurationTest/ConfigurationTest.cpp

```diff
@@ -14,185 +14,237 @@
-        }
-
-        TEST_METHOD(should_instrument_WcfSvcHost)
-        {
-            Configuration configuration(true);
-            Assert::IsTrue(configuration.ShouldInstrument(L"WcfSvcHost.exe", L"", L"foo", L"", false));
-        }
-
-        TEST_METHOD(should_not_instrument_SMSvcHost)
-        {
-            Configuration configuration(true);
-            Assert::IsFalse(configuration.ShouldInstrument(L"SMSvcHost.exe", L"", L"foo", L"", false));
-        }
-
-        TEST_METHOD(should_not_instrument_if_disabled)
-        {
-            Configuration configuration(false);
-            Assert::IsFalse(configuration.ShouldInstrument(L"w3wp.exe", L"", L"foo", L"", false));
-        }
-
-        TEST_METHOD(should_not_instrument_process)
-        {
-            Configuration configuration(true);
-            Assert::IsFalse(configuration.ShouldInstrument(L"foo.exe", L"", L"", L"", false));
-        }
-
-        TEST_METHOD(instrument_process)
-        {
-            ProcessesPtr processes(new Processes());
-            processes->emplace(L"foo.exe");
-            Configuration configuration(true, Logger::Level::LEVEL_INFO, processes);
-            Assert::IsTrue(configuration.ShouldInstrument(L"foo.exe", L"", L"", L"", false));
-        }
-
-        TEST_METHOD(global_xml_agent_enabled_missing_local_xml_agent_enabled_missing)
-        {
-            Configuration configuration(_missingAgentEnabledXml, _missingAgentEnabledConfigPair);
-            Assert::IsTrue(configuration.IsAgentEnabled());
-        }
-
-        TEST_METHOD(global_xml_agent_enabled_missing_local_xml_agent_enabled_true) {
-            Configuration configuration(_missingAgentEnabledXml, _agentEnabledPair);
-            Assert::IsTrue(configuration.IsAgentEnabled());
-        }
-
-        TEST_METHOD(global_xml_agent_enabled_missing_local_xml_agent_enabled_false) {
-            Configuration configuration(_missingAgentEnabledXml, _agentDisabledPair);
-            Assert::IsFalse(configuration.IsAgentEnabled());
-        }
-
-        TEST_METHOD(global_xml_missing) {
-            std::function<void(void)> func = [this]() {
-                Configuration configuration(_noXml, _agentDisabledPair);
-            };
-
-            Assert::ExpectException<NewRelic::Profiler::Configuration::ConfigurationException>(func, L"Should throw configuration exception when there's no global config.");
-        }
-
-        TEST_METHOD(global_xml_agent_enabled_false_local_xml_agent_enabled_false) {
-            Configuration configuration(_agentDisabledXml , _agentDisabledPair);
-            Assert::IsFalse(configuration.IsAgentEnabled());
-        }
-
-        TEST_METHOD(global_xml_agent_enabled_false_local_xml_agent_enabled_missing)
-        {
-            Configuration configuration(_agentDisabledXml, _missingAgentEnabledConfigPair);
-            Assert::IsFalse(configuration.IsAgentEnabled());
-        }
+namespace NewRelic {
```

Comment:
I know VisualStudio does this automatically, but the style convention we typically follow is to have all the namespaces on a single line. I don't know the history behind it though.

Reply:
Yeah, I'm (unfortunately) a serial file reformatter. Wish there was a way to tell VS how we wanted this stuff. I can see about reverting, though it's a bit obnoxious and a pretty artibrary format... ;-)